static/src/javascripts/projects/common/views/acquisitions-epic-thank-you.html

```diff
@@ -0,0 +1,14 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            Thank you …
+        </h2>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            … for supporting us. Your contribution and the similar pledges of hundreds of thousands of readers around
+            the world enables our journalists to cover important issues like this year’s elections.
+            And your knowledge and experience makes our reporting better too.
+            Did you know we publish articles and podcasts for our supporters, featuring your views and voices?
+            <a href="<%=membershipUrl%>" target="_blank" class="u-underline">becoming a monthly supporter</a>
```

Comment:
This doesn't seem to match the copy in the image?

Should the text instead be:

> You can learn more about how to get involved here

??? 🤔